Say Their Names/Source/View/Home/Person/PersonPhotoCell.swift

```diff
@@ -8,19 +8,63 @@
 
 import UIKit
 
-class PersonPhotoCell: UICollectionViewCell {
-
-    override func awakeFromNib() {
-        super.awakeFromNib()
-        // Initialization code
+final class PersonPhotoCell: UICollectionViewCell {
+    
+    var personImage: UIImage? {
+        didSet {
+            DispatchQueue.main.async {
+                self.personImageView.image = self.personImage
+            }
+        }
+    }
+    
+    private let personImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.contentMode = .scaleAspectFill
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        return imageView
+    }()
+    
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        personImageView.image = nil
+    }
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+    
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
     }
+    
+    private func setup() {
+        addSubview(personImageView)
+        
+        personImageView.topAnchor.constraint(equalTo: topAnchor).isActive = true
```

Comment:
This one ran away from the rest

Reply:
Nice catch! I was doing some testing and forgot to get it back into the array